apps/rush-lib/src/pluginFramework/RushLifeCycle.ts

```diff
@@ -1,14 +1,78 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
-import { AsyncSeriesHook } from 'tapable';
+import { AsyncSeriesHook, HookMap } from 'tapable';
 
 /**
+ * Information about the currently executing action provided to plugins.
+ * @beta
+ */
+export interface IRushAction {
```

Comment:
Is `Action` the phrasing we want to go with here? That may confuse with Heft actions.

Reply:
Considering that I called them commands in the hook names, will update

Reply:
Though they mean the same thing here as in Heft